functions/src/typings.ts

```diff
@@ -47,6 +47,8 @@ export const enum AUTHOR_ASSOCIATION {
 
 export interface CachedPullRequest extends Github.PullRequestsGetResponse {
   pendingReviews?: number;
+  mergeable_state?: string;
+  draft?: boolean;
```

Comment:
can you add docs for these states?  where do these states come from and why are they part of CachedPullRequest rather than PullRequestsGetResponse?

Am I missing something or are you just satisfying the compiler so that it doesn't complain about new properties due to outdated `Github.PullRequestsGetResponse` typings?

Reply:
Yes, it's new properties that weren't available at the time.
Probot has been updated and those typings as well, we need to update the libraries to have those.